systemd.yaml

```diff
@@ -114,6 +114,27 @@ subpackages:
             kernel-install --version
             kernel-install --help
 
+  - name: "udev"
+    description: "/dev hotplug management daemon"
+    dependencies:
+      runtime:
+        - libsystemd-shared
+    pipeline:
+      - runs: |
+          cd ${{targets.destdir}}/
+          find . -name '*udev*' -exec cp -v --parents -r '{}' ${{targets.subpkgdir}} \;
+          find . -name '*systemd-hwdb*' -exec cp -v --parents -r '{}' ${{targets.subpkgdir}} \;
+
+          find . -name '*udev*' -exec rm -f '{}' \;
+          find . -name '*systemd-hwdb*' -exec rm -f '{}' \;
+    test:
+      pipeline:
+        - runs: |
```

Comment:
we can add `systemd-hwdb update -s` to the tests, it seems it does not rely on running udev/systemd 
Also we can do a generic query and check if output is valid:

```console
systemd-hwdb query pci:v00001234d00007000sv00001AF4sd00001100bc06sc01i00
ID_PCI_CLASS_FROM_DATABASE=Bridge
ID_PCI_SUBCLASS_FROM_DATABASE=ISA bridge
```

for example

Reply:
> we can add `systemd-hwdb update -s` to the tests, it seems it does not rely on running udev/systemd Also we can do a generic query and check if output is valid:
> 

looks done below, thanks @justinvreeland